Add vlcVersion and vlcChangeset props to get the libVLC runtime version

asherkin opened the pull request:
This adds them to both to the module exports (to get the version without actually initializing libVLC) and the returned player instance (more useful for wcjs-renderer users).

RSATom commented on src/JsVlcPlayer.cpp:
```diff
@@ -204,6 +207,8 @@ void JsVlcPlayer::initJsApi( const v8::Handle<v8::Object>& exports )
     Local<Function> constructor = constructorTemplate->GetFunction();
     _jsConstructor.Reset( isolate, constructor );
+    SET_RO_PROPERTY( exports, "vlcVersion", &JsVlcPlayer::vlcVersion );
+    SET_RO_PROPERTY( exports, "vlcChangeset", &JsVlcPlayer::vlcChangeset );
```
This is not correct, since there are no C++ class instance attached to `exports`. I had to add some sort of check for it in  `SET_RO_PROPERTY`...
And maybe even better choice will be just set `version` and `changeset` as string constants, since it could not be changed during one session... I'll think how to do it better.

RSATom commented:
@asherkin, Your solution have some issues, but I will fix it. Thank you!

asherkin commented:
Whoops! Who needs a valid thisptr anyway!
I've pushed take 2, which hopefully addresses these concerns.

The player proto gets proper constants (not sure on the name here, would you prefer capitalized?), and the exports gets `ForceSet`'d props (no idea how replacing its proto would go).
The idea behind prefixing them with `vlc` was to possibly expose WC.js version info as well in the future, but it may make more sense to prefix *that* instead.

RSATom commented:
Applied, Thanks!
